earth_enterprise/src/fusion/rasterfuse/gepackgen.cpp

```diff
@@ -63,6 +66,7 @@ int main(int argc, char *argv[]) {
   options.flagOpt("?", help);
   options.opt("config", configFile,  khGetopt::FileExists);
   options.opt("output", output);
+  options.opt("decimation", decimation_threshold);
```

Comment:
would be good to have some range validator like we have for numcpus.

Reply:
Can't until we decide on what a valid range might be . Lee discussed making that into a separate task.